src/rpc/rawtransaction.cpp

```diff
@@ -974,6 +974,8 @@ static RPCHelpMan testmempoolaccept()
             continue;
         }
         const auto& tx_result = it->second;
+        // Package testmempoolaccept doesn't allow transactions to already be in the mempool.
+        CHECK_NONFATAL(tx_result.m_result_type != MempoolAcceptResult::ResultType::MEMPOOL_ENTRY);
```

Comment:
How is this enforced? More specifically, what happens if `testmempoolaccept` is called with multiple transactions where one or more is already in the mempool? It looks to me like the assert in `PackageMempoolChecks()` could be triggered.

Reply:
testmempoolaccept doesn't allow already-in-mempool transactions, so it will be rejected